lib/bolt/config.rb

```diff
@@ -171,6 +172,12 @@ def update_from_file(data)
           self[:transports][:pcp][:"task-environment"] = data['pcp']['task-environment']
         end
       end
+
+      if data['local']
```

Comment:
I'd like to pull some of this transport-specific knowledge out of the Config object into the transports themselves eventually.